server/plugin/command.go

```diff
@@ -44,21 +46,26 @@ func validateFeatures(features []string) (bool, []string) {
 	valid := true
 	invalidFeatures := []string{}
 	hasLabel := false
+	isRepo := false
 	for _, f := range features {
 		if _, ok := validFeatures[f]; ok {
 			continue
 		}
-		if strings.HasPrefix(f, "label") {
+		if strings.Contains(f, "label") {
```

Comment:
Specifically for this test case: https://github.com/mattermost/mattermost-plugin-github/pull/380/files#diff-05e1a5ecb4a0133c535cadfdf50ce3daR75-R84

I'm open to suggestions, but I think we need to use Contains() here because otherwise when a test passes something like `{"subscribe", "dhaussermann/hello-world", issues,label:"help"}` it won't pick up on the label.

Reply:
In theory, the features will be separated by commas, so in the example you are giving, f will be `issues` or `label:"help"`. In case of `label:"help"` the HasPrefix should still work. Am I right?

Reply:
I can try to split the arguments into different args, but from what I saw in the test the argument was `issues,label:help` as one argument. I can split that argument into two different args effectively making it `issues` then `label:help` which will allow us to use `HasPrefix()`. 

I just didn't want to alter the test too much just in case it was catching some particular functionality. 

Let me know! If you think that's better, I can absolutely change it! 🙂

Reply:
I changed the test locally, if we think this is the approach we want to take let me know and I can push it up!

Reply:
After a second look, many of the things you are checking here probably should not go here. The repo name should never reach the validateFeatures function. It will be handled in "handleSubscribesAdd", and only the "option list" will be passed to the validate features function. That is the main reason why the tests needed some tweaking too.

Reply:
Ah! I'll take another look then. Thank you for explaining that to me!